docs/_content/standaard/verzoeken/index.md

```diff
@@ -0,0 +1,67 @@
+
+# Redenen voor het ontwikkelen van een Verzoeken API
+Het (her)introduceren van verzoeken in het zaakgericht werken biedt een aantal voordelen, onder andere op de volgende gebieden:
+* Compleetheid: we standaardiseren een extra deel van het proces dat leidt tot levering van producten en diensten, namelijk (het generieke deel van) de gegevens die nodig zijn voor het aanvragen hiervan. Hiermee kunnen we (het generieke deel van) StUF-EF vervangen.
+* Herkenbaarheid: de klant communiceert met de gemeente niet langer alleen over zaken (een voor hem niet altijd herkenbaar concept), maar over verzoeken, die (als verzoeken correct worden gebruikt) een voor de klant herkenbare (combinatie van) product(en) vertegenwoordigen. Het blijft mogelijk de voortgang van de afhandeling van een zaak terug te koppelen naar de klant.
+* Flexibeler (1): gemeenten worden flexibeler in het aanbieden van meerdere gerelateerde producten tegelijk (één verzoek kan leiden tot meerdere zaken). Tegelijkertijd kunnen meerdere verzoeken worden afgehandeld in één zaak.
+* Flexibeler (2): bij het indienen van een verzoek hoeft nog niet duidelijk te zijn hoe dat verzoek zal worden behandeld. In de huidige praktijk moet een vraag om levering van een product of dienst immers direct te worden omgezet naar een zaak van een bepaald zaaktype, anders kan immers geen identificerend kenmerk worden teruggestuurd aan de klant. Door verzoeken onderdeel te maken van het zaakgericht werken, kan het verzoeknummer aan de klant gecommuniceerd (‘uw verzoek is ontvangen!’), zonder dat daarmee is vastgelegd op welke manier de behandeling plaatsvindt (en wellicht is in sommige gevallen een zaak/ontvangstbevestiging niet nodig omdat een verzoek direct geautomatiseerd kan worden afgehandeld). Dit betekent dus ook dat een verzoek op een niet-zaakgerichte manier kan worden afgehandeld!
+* Zuiverheid: er ontstaat een duidelijk onderscheid tussen 'klantorders' en interne 'productie-/pickorders'.
+* Verantwoording: de informatie die bij de intake aanwezig was, is de informatie die hoort bij het verzoek. Daarmee is meteen duidelijk dat informatie die aan een zaak (of een andere behandelwijze) is gekoppeld, na de initiële aanvraag is toegevoegd.
+
+# Intake
+Op een verzoek wordt een intake gedaan. Daarbij wordt het verzoek niet inhoudelijk beoordeeld. Wel wordt gekeken of de aangevraagde producten geleverd kunnen worden, en of alle daarvoor benodigde gegevens zijn ingevuld. Na de intake is het verzoek 'in behandeling genomen' danwel 'buiten behandeling gesteld'. De status van een verzoek geeft weer wat er met het verzoek in de intake is gedaan. De status kan in de PIP getoond worden, maar let op: dit kan verwarrend zijn als ook statussen van zaken getoond worden. Het is aan consumers om dit netjes te presenteren.
```

Comment:
Om te kunnen controleren of de benodigde gegevens zijn ingevuld, wil je ook vooraf kunnen controleren wat de (minimale) benodigde gegevens zijn. Als je (zoals gesteld bij regel 44) typen verzoeken uit de PDC haalt, zou dat informatiemodel uitgebreid moeten worden met verplichte gegevens. Al helemaal als het mogelijk moet zijn om (zoals gesteld bij regel 20) automatische intake mogelijk moet worden.